components/Alerts/alerts.css

```diff
@@ -0,0 +1,60 @@
+@import '/css/main.css';
+
+.alert {
+    display: flex;
+    height:70% ;
+    width: 90%;
+    flex-direction: row;
+    align-items: center;
+    flex-wrap: wrap;
+    justify-content: flex-start;
```

Comment:
You can remove justify-content. I think it is not used.

Reply:
yes you are right ,It was unused therefore I have removed it